src/app.rs

```diff
@@ -161,6 +161,36 @@ impl SsbhApp {
         // TODO: Reset selected indices?
         // TODO: Is there an easy way to write this?
     }
+
+    pub fn hide_expressions(&mut self) {
+        let patterns: [&str; 34] =
+            ["blink", "attack", "ouch", "talk",
+            "capture", "ottotto", "escape", "half",
+            "pattern", "result", "harf", "hot", "heavy",
+            "voice", "fura", "catch", "cliff", "flip",
+            "bound", "down", "final", "result", "steppose",
+            "sorori", "fall", "appeal", "damage", "camerahit", "laugh",
+            "breath", "swell", "_low", "_bink", "inkmesh"];
+
+        for (i, folder) in self.models.iter().enumerate() {
+            if let Some(render_model) = self.render_models.get_mut(i) {
+                for mesh in &mut render_model.meshes {
+                    let name: &str = &mesh.name.to_lowercase();
```

Comment:
Rust can infer the `&str`. You actually don't need the reference at all, so this can just be `let name = mesh.name.to_lowercase();`